include/picongpu/param/fieldAbsorber.param

```diff
@@ -35,7 +35,7 @@ namespace picongpu
         namespace absorber
         {
             // This variable is not required, defined for convenience
-            constexpr uint32_t THICKNESS = 32;
+            constexpr uint32_t THICKNESS = 12;
```

Comment:
Why do you change the default absorber thickness?

Reply:
So that it corresponds to the recommended value for default absorber. For exponential we recommend 32, for PML we recommend 8-12